.circleci/config.yml

```diff
@@ -1,43 +1,221 @@
 version: 2.1
 
+orb_promotion_filters: &orb_promotion_filters
+  branches:
+    ignore: /.*/
+  tags:
+    only: /^(major|minor|patch)-release-v\d+\.\d+\.\d+$/
```

Comment:
how do we know the version before it is promoted?

Reply:
@eddiewebb The version numbering must be manually derived based on the last published number. The numbering used in the tag is actually irrelevant to the CLI promotion command, but my thoughts were that this tagging pattern might be useful for recordkeeping purposes (e.g. for those who use GitHub releases) and also keeps the tags unique.

Reply:
got it, thanks